python/table.py

```diff
@@ -0,0 +1,51 @@
+def table(dataset):
```

Comment:
Did you consider not hand-rolling this as a string? i.e use https://pypi.org/project/htmlBuilder/, http://www.yattag.org/ maybe others?

Reply:
Yes I did, but this was faster as a quick hack for now. It should probably be extended anyway to list 0-D variables separately, and deal with bin-edges, etc.. I will add this to the TODO list.